### ISSUE TITLE
Unable to import the module in Angular 2

### ISSUE THREAD

**Issue by chaitanyya** [FineUploader/fine-uploader#1839]
I tried importing the module in my angular 2 component using
`import qq from 'fine-uploader';`
and 
`import { qq } from 'fine-uploader';`

It's giving me the following error:
`/node_modules/fine-uploader/typescript/fine-uploader.d.ts is not a module.`

I'm using version "5.14.2". I installed it using npm.

**Comment by chaitanyya** [comment]
Okay, I finally figured it out. It'll be great if you add some similar process in your documentation. This is what I did to make it work:

**Step One:**
Add `/// <reference types="fine-uploader" />` on top of you component file.

**Step Two:**
Declare following variables:
```
uploader: FineUploader.qq;
uiOptions: FineUploader.UIOptions;
coreEvents: FineUploader.CoreEvents;
```

**Step Three:**
Add UI options and initialise FineUploader in `ngOnInit`
```
this.uiOptions = {
    element: document.getElementById('qq-template'),
    template: "qq-template"
};
this.uploader = new qq.FineUploader(this.uiOptions);
```

**Step Four:**
Add the template HTML in your component.html file. You may find the HTML in `node_modules/fine-uploader/fine-uploader/templates/`

**Comment by SinghSukhdeep** [comment]
@chaitanyya yes this is the correct way of doing it as of now.
Have a look at [here](https://github.com/FineUploader/fine-uploader/blob/develop/client/typescript/fine-uploader.test.ts) for more info on syntax.
Sorry about the documentation, whole TS integration details aren't documented properly.

**Comment by SinghSukhdeep** [comment]
This is fixed in #1840 and released in 5.15.0
